src/databricks/labs/remorph/reconcile/constants.py

```diff
@@ -71,7 +71,7 @@ class ThresholdMatchType(AutoName):
 
 
 class Constants:
```

Comment:
this needs a whole revamp, please raise a issue as techdebt

Reply:
#313